README.md

```diff
@@ -837,6 +837,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Jessica Teo](https://github.com/jessicateocw)
   - [Jessie Auguste](https://github.com/jessiebelle)
   - [JesusGerardoAguiar](https://github.com/JesusGerardoAguiar)
+  - [Jeferson Cardoso](https://github.com/cardosource)
```

Comment:
Please edit you name and insert it in line 833. Names should be in alphabetical order.